common/test/acceptance/tests/lms/test_account_settings.py

```diff
@@ -221,11 +231,13 @@ def test_gender_field(self):
             u'',
             [u'Female', u''],
         )
+        self.assert_event_emitted_num_times('edx.user.settings.changed', self.start_time, self.user_id, 2)
 
     def test_year_of_birth_field(self):
         """
         Test behaviour of "Year of Birth" field.
         """
+        # Note that when we clear the year_of_birth here we're firing an event.
```

Comment:
Fair enough. I considered that, but I prefer the idea of making sure that event is fired.